src/corelibs/U2View/src/ov_sequence/AnnotatedDNAView.cpp

```diff
@@ -966,6 +966,10 @@ void AnnotatedDNAView::unregisterSplitWidget(ADVSplitWidget *splitWidget) {
     splitWidgets.removeOne(splitWidget);
 }
 
+QList<ADVSplitWidget *> AnnotatedDNAView::getSplitWidgets() {
```

Comment:
This method is not needed. You can lookup any widget from GUI tests by object name.

Reply:
done